olive/auto_optimizer/config_template/pass_capability.yaml

```diff
@@ -70,3 +70,12 @@ OrtPerfTuning:
   EP: null
   precision: null
   accelerator: null
+NVModelOptQuantization:
```

Comment:
@jambayk how this capability is used? ModelOpt work both with CPU and CUDA EP. But I didn't understand the `accelerator` field.

Reply:
Olive has a feature where it uses this yaml file to automatically create model optimization workflows by mixing and matching compatible passes. 
The accelerator field refers to the compute hardware device. Olive uses a combination of ep,accelerator to categorize workflows. so the eps and accelerators here are used to create a matrix of supported combos.

I think you can remove this pass from here. Since the pass does not support data configs yet, the auto optimizer would not be able to use this pass as it won't have the dataloader func to provide to it. 
@trajepl can confirm?

Reply:
I will remove for now. We can add them back later if needed.